webapi/Options/PlannerOptions.cs

```diff
@@ -11,20 +11,22 @@ namespace CopilotChat.WebApi.Options;
 public class PlannerOptions
 {
     /// <summary>
-    /// Whether to allow missing functions in plan on creation. If allowed, proposed plan will be sanitized of no-op functions.
-    /// Functions are considered missing if they're not available in the planner's kernel's context.
+    /// Options to handle planner errors.
     /// </summary>
-    public class MissingFunctionErrorOptions
+    public class PlannerErrorOptions
```

Comment:
Planner is redundant here since the containing class is PlannerOptions

```suggestion
    public class ErrorOptions
```